lib/mllp/tcp.ex

```diff
@@ -42,8 +42,7 @@ defmodule MLLP.TCP do
   @spec is_closed?(socket :: :gen_tcp.socket()) :: boolean()
   def is_closed?(socket) do
     case recv(socket, _length = 0, _timeout = 1) do
-      {:error, :closed} -> true
-      {:error, :enotconn} -> true
+      {:error, reason} -> if reason == :timeout, do: false, else: true
```

Comment:
@mfos239  It might make more sense to pattern match on the {:ok, _} case first so that the success case is explicit rather than implicit, and then log the error cases reason for visibility. 

That would give us total confidence that 'is_closed?() == false' is accurate and there's no possibility for a false indication, as well as giving us some visibility as to why it's evaluated as closed if that's the case. 

What do you think?